projects/chintai/index.js

```diff
@@ -1,19 +1,178 @@
+        "BaNDYFKnVsJk9aS8zRifaZ3Duc312PPL3obZVWJNDeSr",
+        "7KjuZf6Dwtd9JJjEnzvrb5LM2iJKvVLa47edrBE11am6",
+      ],
+      meteoraAccounts: [
+        "5qmWvgyCSZQNJs2NWwTKdFsmcRT2NJJAuomti8GBCBfm",
+        "Gs1dVEjVT3mSBKFxLsGw2BtQnkKaGKFkPc4CwchtqvP7",
+        "5qmWvgyCSZQNJs2NWwTKdFsmcRT2NJJAuomti8GBCBfm",
+        "8y3jryTeRcerNRyozfQR5bgdjME24hqBMFLQptZkBEhb",
+        "LqZmiW9gmtNnHMu56stPii414pT5EUwhXkpiwa6d4RW",
+        "FhCPZa4bjZdaV9GBrvjyeeK95XmjkU83AfwTNdr7bjn3",
+      ],
+      raydiumAccounts: [
+        "3yc8oB6RE5Rz8ThRTJSvzwFqZSMpNAzzwMtJifgn2m7Z",
+        "6yDMv6GEgMsJWV6Fr7Lk8YfXcMEAvz18CRsPzVn6B4hB",
+        "2qmJC3SVFSkK2fYz7fswkPn7ySqtf9xH2AkLBZ999sbr",
+        "AD7qRvY14byTLY19tWZTvXLaMF7AZj4dw4Gb1nHbgaW6",
+      ],
+    },
+  },
+};
+
+async function solanaTvl() {
+  return sumTokens2({
+    tokenAccounts: [
+      config.solana.pools.orcaTokenAccounts,
+      config.solana.pools.meteoraAccounts,
+      config.solana.pools.raydiumAccounts,
+    ].flat(),
+  });
+}
+
+async function eosTvl() {
+  const tokens = [
     ["chexchexchex", "CHEX", "chex-token"],
-    ["eosio.token", "EOS", "eos"]
-];
+    ["eosio.token", "EOS", "eos"],
+  ];
+  return await get_account_tvl(["swap.defi", "newdexpublic"], tokens);
+}
+
+const scaleValue = (value, times = 1) =>
+  BigNumber(value).times(times).toFixed(0);
 
-// Chintai
-// https://chintai.io/
-async function eos() {
-  return await get_account_tvl("chintailease", tokens);
+async function toBalances(symbol, value) {
+  const address = ADDRESSES.ethereum[symbol];
+  if (!address) {
+    console.error("Could not find address for:", symbol);
+    return null;
+  }
+
+  const decimals = (
+    await sdk.api.abi.call({
+      target: address,
+      abi: "erc20:decimals",
+      chain: "ethereum",
+    })
+  ).output;
+
+  return {
+    [address]: scaleValue(value, 10 ** decimals),
+  };
+}
+
+async function chintaiTvl() {
+  const stats = await get("https://sg.app.chintai.io/api/stats");
+
+  const balances = {};
+  for (const [symbol, supply] of Object.entries(stats.tokenBalances)) {
+    const balance = await toBalances(symbol, supply);
+    if (balance) {
+      Object.assign(balances, balance);
+    } else {
+      const coinGeckoId = chintaiSymbolToCoinGeckoIds[symbol];
+      if (!coinGeckoId) {
+        console.error("Could not find CoinGecko ID for:", symbol);
+        continue;
+      }
+      await sdk.util.sumSingleBalance(balances, coinGeckoId, supply, "");
+    }
+  }
+
+  return balances;
 }
 
 module.exports = {
-  methodology: `Chintai TVL is achieved by querying token balances from Chintai leasing smart contract.`,
+  methodology:
+    "Chintai TVL is achieved by querying the USD value of all the asset issued and staked on the Chintai platform as well as the value locked in all the CHEX token liquidity pools.",
+  ethereum: {
+    tvl: sumTokensExport({
+      chain: "ethereum",
+      owners: config.ethereum.pools.uniswapPools,
```

Comment:
value on tokens in uniswap pools belong to uniswap tvl, not your protocol

Reply:
same for the other chains

Reply:
I'll remove all the pools